README.md

```diff
@@ -17,3 +17,17 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 ## (optional) Pausing comments
 NEXT_PUBLIC_COMMENTS_ARE_PAUSED=true
 ```
+
+## Authorization configuration
+
+Create file **.env.local** and fill it with information from [Auth0 Application Profile](https://manage.auth0.com/dashboard):
+
+```ini
+AUTH0_SECRET=replace-with-your-own-secret-generated-with-openssl
+AUTH0_BASE_URL=http://localhost:3000
+AUTH0_ISSUER_BASE_URL='https://{DOMAIN}'
```

Comment:
Any way we can configure auth0 for Netlify preview releases? I wonder if there is some recipe we can find.

Reply:
Meanwhile, I’m getting this when trying to press login at  https://deploy-preview-7--dtp-stat-on-nextjs.netlify.app/

Reply:
Found and tested